tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js

```diff
@@ -143,6 +143,11 @@ describe("Feature: Repeating Sections with Hub and Spoke", () => {
       expect(browser.getUrl()).to.contain("/sections/personal-details-section");
     });
 
+    it("When the user views the summary for a repeating section, Then the custom page title is shown", () => {
```

Comment:
This is now no longer testing for a custom page title.

Reply:
changing to `Then the page title is shown`

Reply:
I think the test can be removed. It doesn't make sense for it to be added as part of this work.

Reply:
I don't agree. We should functionally test these page titles (esp. since I have resolved a number of issues with them). This change is too small to warrant a whole separate card, and I doubt we can prioritise a card to review testing of page titles in functional tests at the moment.